DFE-Digital/Dfe.Academies.External
# 111384 upload previous financial year docs

Resolves [#111384](https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_boards/board/t/Apply%20to%20become/Stories/?workitem=111384)

## Type of change


## Expected behaviour


two new document uploads on the schools previous financial year. Validation changes in line with criteria so either a file or description must be provided or both



## Steps to reproduce issue (if relevant)


1.
2.
3.
4.

## Steps to test this PR

1.Upload files on previous financial year page 
2.Check they arrive in SharePoint with the right prefix in the right folder
3.Return to the previous financial year page and remove file
4. File should be removed from SharePoint

## Prerequisites
